tools/ingest_historical_csv.py

```diff
@@ -92,15 +92,16 @@ def _dedupe_data(df: pd.DataFrame) -> pd.DataFrame:
 
 
 def _update_df(df: pd.DataFrame) -> pd.DataFrame:
-    df['ecosystem'] = df['ecosystem'].str.upper()
-    df['status'] = df.apply(lambda x: _get_status_type(x['status']), axis=1)
+    if len(df) != 0:
+        df['ecosystem'] = df['ecosystem'].str.upper()
+        df['status'] = df.apply(lambda x: _get_status_type(x['status']), axis=1)
```

Comment:
Does this not give a warning?

Reply:
Warning should read something like "assignment to a view of the data".

Reply:
not getting any warning